### PR TITLE
Implemented endpoint to remove a torrent from a channel

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@lfdversluis can you review?

**Comment by lfdversluis** [comment]
Looks good

**Comment by devos50** [comment]
@whirm can you review?

File: Tribler/Core/Modules/restapi/channels/channels_torrents_endpoint.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 from Tribler.Core.exceptions import DuplicateTorrentFileError
 
 
+UNKNOWN_TORRENT_MSG = "this torrent is not found in the specified channel"
+UNKNOWN_COMMUNITY_MSG = "the community for the specific channel cannot be found"
```
Comment (whirm) [inline Tribler/Core/Modules/restapi/channels/channels_torrents_endpoint.py]: specified

**Comment by whirm** [comment]
other than the typo, looks fine.

Ffix and squash!

**Comment by devos50** [comment]
@whirm fixed + squashed

**Comment by devos50** [comment]
@whirm mac timing out due to credit mining test. Want to retest?

**Comment by whirm** [comment]
@devos50 please make a PR disabling the misbehaving tests. @ardhipoetra can enable them on his bugfix PR once they are rock solid.

**Comment by devos50** [comment]
@whirm see #2382

**Pull request merged**